mixer/adapter/stackdriver/config/config.proto

```diff
@@ -40,223 +40,177 @@ option (gogoproto.gostring_all) = false;
 
 // Configuration format for the `stackdriver` adapter.
 message Params {
-    // Endpoint URL to send stackdriver data - leave empty to use the
-    // StackDriver SDK's default value (monitoring.googleapis.com).
-    string endpoint = 1;
-
-    // GCP Project to attach metrics to.
-    string project_id = 2;
-
-    // This adapter batches the data it sends to Stackdriver; we will push to stackdriver every push_interval.
-    // If no value is provided we default to once per minute.
-    google.protobuf.Duration push_interval = 3 [(gogoproto.nullable)=false, (gogoproto.stdduration) = true];
-
-    // Credentials must be attached to every call made to StackDriver. This
-    // can be an API key, a service account secret key, or Oauth2 tokens.
-    //
-    // TODO: do we want to support oauth?
-    oneof creds {
```

Comment:
By removing this, are we insisting then that SD adapters only be configured on GCE/GKE installs?  How do non-GCE/GKE handlers attach service acct creds without such a place to hang them?